Library/Formula/squid-deb-proxy.rb

```diff
@@ -0,0 +1,174 @@
+class SquidDebProxy < Formula
+  desc "APT repo caching proxy based on Squid."
+  homepage "https://launchpad.net/squid-deb-proxy"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/s/squid-deb-proxy/squid-deb-proxy_0.8.14.tar.gz"
+  sha256 "d30aa9d8185c1e424608deb307f8e92aee8270acccc3c0b26af1d03a2feae92b"
+
+  depends_on "squid"
+
+  # assorted changes that should be submitted upstream
+  patch :DATA
+
+  def install
+    # fix up issues specific to Homebrew
+    inreplace "squid-deb-proxy.conf" do |s|
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! %r{([\s='"])/etc/}, "\\1#{etc}/"
+      s.gsub! %r{([\s='"])/var/}, "\\1#{var}/"
+    end
+
+    inreplace "init-common.sh" do |s|
+      # install should be as the calling user
+      s.gsub! /(\binstall\b.*?)(\s+-[og]\s*\S+)+/, "\\1"
+
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! "/usr/sbin/squid", "#{sbin}/squid"
+      s.gsub! %r{([\s='"])/etc/}, "\\1#{etc}/"
+      s.gsub! %r{([\s='"])/var/}, "\\1#{var}/"
+    end
+
+    # pre-installation from debian/rules
+    # we take both sets because we're not either distro
+    IO.write(
+      "mirror-dstdomain.acl",
+      IO.read("mirror-dstdomain.acl.Debian") +
+        IO.read("mirror-dstdomain.acl.Ubuntu")
+    )
+
+    system "make", "install", "DESTDIR=#{prefix}"
+
+    # additional installation from debian/squid-deb-proxy.install
+    mkdir_p "#{prefix}/usr/share/squid-deb-proxy"
+    cp "init-common.sh", "#{prefix}/usr/share/squid-deb-proxy/"
```

Comment:
> This is a bit odd. What is it doing?

For some reason the upstream project, which is intended to be a Debian package, installs this file using the Debian packaging metadata instead of the `install` target of the Makefile. I therefore have to install it explicitly.